cmd/installer/cli/install2.go

```diff
@@ -483,9 +491,21 @@ func recordInstallation(ctx context.Context, flags Install2CmdFlags, k0sCfg *k0s
 		},
 	}
 	if err := kubeutils.CreateInstallation(ctx, kcli, &installation); err != nil {
-		return fmt.Errorf("create installation: %w", err)
+		return nil, fmt.Errorf("create installation: %w", err)
 	}
 
+	return &installation, nil
+}
+
+func updateInstallation(ctx context.Context, install *ecv1beta1.Installation) error {
+	kcli, err := kubeutils.KubeClient()
+	if err != nil {
+		return fmt.Errorf("create kube client: %w", err)
+	}
+
+	if err := kubeutils.UpdateInstallation(ctx, kcli, install); err != nil {
+		return fmt.Errorf("Failed to update installation")
```

Comment:
i know the messaging will be weird to the end user but lets just keep this consistent

```suggestion
		return fmt.Errorf("update installation")
```